beacon-chain/utils/shuffle.go

```diff
@@ -2,71 +2,24 @@
 package utils
 
 import (
-	"errors"
+	"encoding/binary"
 	"fmt"
 
-	"github.com/ethereum/go-ethereum/common"
 	"github.com/prysmaticlabs/prysm/shared/bytesutil"
 	"github.com/prysmaticlabs/prysm/shared/hashutil"
 	"github.com/prysmaticlabs/prysm/shared/params"
 )
 
+// HashFn sets the current hash function
```

Comment:
```suggestion
// HashFn sets the current hash function.
```

Reply:
its not used, i removed it

Reply:
Huh. I still see it. We should add a period at the end